bitnami/external-dns/templates/_helpers.tpl

```diff
@@ -308,10 +308,14 @@ Compile all warnings into a single message, and call fail.
 {{- $messages := without $messages "" -}}
 {{- $message := join "\n" $messages -}}
 
+{{- if .Values.validation.enabled -}}
 {{- if $message -}}
 {{-   printf "\nVALUES VALIDATION:\n%s" $message | fail -}}
 {{- end -}}
 {{- end -}}
+{{- end -}}
+
+{{/*
```

Comment:
Is this a typo? These lines shouldn't be needed right?

Reply:
Removed!